app/js/data/pool.js

```diff
@@ -63,7 +62,10 @@ class Pool {
     throw new Error('All reservations exhausted');
   }
 
-  generate() {
+  // Generate new random card prompts for each slot given filtered contents.
+  generateNew() {
+    this.seed = this.genSeed();
+
     let i = 0;
     for(let slot = 0; slot < 24; slot++) {
```

Comment:
```suggestion
    for(let slot = 0, i = 0; slot < 24; slot++) {
```
Kat says please opportunity fix his scope mistake.